aten/src/ATen/CMakeLists.txt

```diff
@@ -83,8 +83,12 @@ FILE(GLOB native_sparse_hip_cpp "native/sparse/hip/*.cpp")
 FILE(GLOB native_quantized_hip_hip "native/quantized/hip/*.hip")
 FILE(GLOB native_quantized_hip_cpp "native/quantized/hip/*.cpp")
 
+# Mobile
+FILE(GLOB native_mobile_cpp "native/mobile/internal/*.cpp")
+FILE(GLOB native_mobile_cpu_cpp "native/mobile/cpu/internal/*.cpp")
```

Comment:
Should we always build these files, or only in the mobile build?

Reply:
Thanks for point that out.  I will check with Jiakai.  Probably no.

Reply:
I was planning on adding the mobile gpu backend as a separate folder right next to cpu to consolidate all mobile backends together. I agree that neither XNNPACK, nor the GPU backend under work, is mobile specific. I am very open to changing the folder structure to how you guys think is best. Suggestions are welcome.

Reply:
Just put it in `xnnpack`

Reply:
@ezyang @gchanan @ljk53  So I will put these files under aten/src/ATen/native/xnnpack.  Do we have an agreement?